### DISCUSSION TITLE
use camel case in POJO and snake case in graphql. What should I do?

### DISCUSSION THREAD

**Discussion by oonxt** [Netflix/dgs-framework#946] [Q&A] [answered]
I'm trying to use camel case in POJO and snake case in graphql. What should I do?
like this:

the pojo
```
class User {
     UUID id;
     LocalDateTime createdAt;
}
```
the graphql:
```
type User {
    id: UUID
    created_at: DateTime
}
```

**Comment by srinivasankavitha** [comment, marked as answer]
@oonxt - we don't have the ability to do that. The variable names are
directly based on the schema field names.

**Reply by oonxt** [reply]
OK. thx. I got it.
